toolkit/committee-selection/selection/src/ariadne_v2.rs

```diff
@@ -216,10 +218,10 @@ mod tests {
 			r1_count += committee.iter().filter(|c| **c == "R1").count();
 			r2_count += committee.iter().filter(|c| **c == "R2").count();
 		}
-		let tolerance = 5;
-		assert!(1343 - tolerance <= p1_count && p1_count <= 1343 + tolerance);
-		assert!(1220 - tolerance <= r1_count && r1_count <= 1220 + tolerance);
-		assert!(1780 - tolerance <= r2_count && r2_count <= 1780 + tolerance);
+		let tolerance = 50;
+		assert!(13330 - tolerance <= p1_count && p1_count <= 13330 + tolerance);
+		assert!(12000 - tolerance <= r1_count && r1_count <= 12000 + tolerance);
+		assert!(18000 - tolerance <= r2_count && r2_count <= 18000 + tolerance);
```

Comment:
I reverted constants to be "expected values". This was regression in https://github.com/input-output-hk/partner-chains/pull/783/files#diff-aa42176a6aee1bb513e9db2a9c00b5f597e5d78963de2d2c69a2840a3cad5d32R220-R222 where constants ceased to be expected values.